gcpdiag/lint/cloudsql/bp_ext_2023_002_del_protection.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Cloud SQL is not configured with Deletion Protection.
```

Comment:
Something that I didn't notice before, sorry:

The short description should always describe the good state, and not what is failing, so in this case:

Cloud SQL is configured with Deletion Protection

(in general we avoid also punctuation here)